content_sync/management/commands/backpopulate_pipelines.py

```diff
@@ -56,6 +58,13 @@ def add_arguments(self, parser):
             action="store_true",
             help="Unpause the pipelines after creating/updating them",
         )
+        parser.add_argument(
+            "-d",
+            "--delete",
+            dest="delete",
+            action="store_true",
+            help="Delete all existing site pipelines first",
```

Comment:
Could we name this `--delete-all`?

Originally, I was surprised when `backpopulate_pipelines --filter "es-255-physics-of-rock-climbing-spring-2006 --delete-all" deleted all my other pipelines, ie, was unaffected by filter. But that behavior makes sense...otherwise the delete flag wouldn't really do anything (right?).

This is my requested changed, unless `--delete-all` would be inconsistent with our naming for similar behavior elsewhere.

Reply:
Sure, will do